Cargo.toml

```diff
@@ -43,6 +43,7 @@ name = "dalek_benchmarks"
 harness = false
 
 [dependencies]
+cfg-if = "1"
```

Comment:
Note this is already a transitive hard dependency via the `sha2` crate which uses it in a similar capacity.

It's maintained by the core team, so nothing to worry about as far as supply chain attacks go.